### DISCUSSION TITLE
For today, what will you bet?

### DISCUSSION THREAD

**Discussion by Skysafe64** [kyleskom/NBA-Machine-Learning-Sports-Betting#139] [Q&A]
![image](https://user-images.githubusercontent.com/123671403/224524387-e287b424-dc1c-4a0c-90ee-7da2a136dcd9.png)
Everything is in the title :D

**Comment by Marcfeitosa** [comment]
I'm trying to find a way to export that all to excel, including the formats. has anyone done that already

**Reply by betevents** [reply]
you can use xlsxwriter engine for pandas and export dataframes, now it depends how do you want it formatted, one sheet, multiple sheets, columns etc

example from a dataframe:
writer = pd.ExcelWriter('test.xlsx', engine='xlsxwriter')
df.to_excel(writer, sheet_name='Sheet1')
writer.close()

**Comment by Marcfeitosa** [comment]
Great tip
    Em quarta-feira, 15 de março de 2023 às 00:32:28 BRT, betevents ***@***.***> escreveu:  
 
 


you can use xlsxwriter engine for pandas and export dataframes, now it depends how do you want it formatted, one sheet, multiple sheets, columns etc

example from a dataframe:
writer = pd.ExcelWriter('test.xlsx', engine='xlsxwriter')
df.to_excel(writer, sheet_name='Sheet1')
writer.close()
